test/application_system_test_case.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "capybara/rails"
+require "capybara/minitest"
+require "capybara/cuprite"
+
+Ferrum::Browser.new(process_timeout: 5)
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  driven_by :cuprite, using: :chrome, screen_size: [1400, 1400]
```

Comment:
any reasoning behind 1400x1400?

Reply:
Nope, this was default from examples we saw.